.changeset/config.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://unpkg.com/@changesets/config@1.6.0/schema.json",
   "changelog": "@changesets/cli/changelog",
   "commit": false,
-  "linked": [],
+  "linked": [["contracts-bedrock", "contracts-ts"]],
```

Comment:
Link these packages together so they always have the same version

Reply:
This makes sense only while this is mostly generated code.  Should that not become the case this would be less useful